astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -1348,3 +1332,43 @@ def check_wcs(header):
     # Check fall back to scale='utc'
     del hdr["TIMESYS"]
     check_wcs(hdr)
+
+
+################################################################################
+# Tests with Stokes
+################################################################################
+
+
+HEADER_POLARIZED = """
+CTYPE1  = 'HPLT-TAN'
+CTYPE2  = 'HPLN-TAN'
+CTYPE3  = 'STOKES'
+"""
+
+
+@pytest.fixture
+def header_polarized():
+    return Header.fromstring(HEADER_POLARIZED, sep="\n")
+
+
+@pytest.fixture()
+def wcs_polarized(header_polarized):
+    return WCS(header_polarized)
```

Comment:
Unless you were going to add tests with `header_polarized`, I suggest to merge it in here.

(I still prefer just using a class and setting this up with `setup_class`, because that means one can still import and use things without `pytest`, but that is a matter of taste.)

Reply:
I would rather keep it like this as it's how the rest of the file is done, and I want to be consistent and I don't want to change it all :see_no_evil: